packages/rich-list-core/src/controller/AddressParser/dftx/utxosToAccount.ts

```diff
@@ -0,0 +1,13 @@
+import { fromScript } from '@defichain/jellyfish-address'
+import { CUtxosToAccount, DfTx, UtxosToAccount } from '@defichain/jellyfish-transaction'
+import { DfTxAddressParser } from './_abstract'
+
+export class UtxosToAccountParser extends DfTxAddressParser<UtxosToAccount> {
+  OP_CODE: number = CUtxosToAccount.OP_CODE
+
+  extract (accountToAccount: DfTx<UtxosToAccount>): string[] {
+    const addresses = accountToAccount.data.to
```

Comment:
This is for utxosToAccount, which have the structure of

```typescript
export interface UtxosToAccount {
  to: ScriptBalances[] // --------------| n = VarUInt{1-9 bytes}, + n bytes
}
```
I think the `from` field does not exists for utxosToAccount. Do correct me if I am wrong though.
Otherwise, I think its because I forgot to change the naming for params that's why it confused you. I'll make the changes.